parent/pom.xml

```diff
@@ -26,7 +26,7 @@
     <version.jersey2>2.34</version.jersey2>
     <!-- use minimum version of resteasy and jersey -->
     <version.jaxrs.api>2.0.1.Final</version.jaxrs.api>
-    <version.groovy>2.4.21</version.groovy>
+    <version.groovy>4.0.21</version.groovy>
```

Comment:
🔧 Latest available version is `4.0.22`

Reply:
They released it end of last week, updated.